src/sql/parts/notebook/cellViews/textCell.component.ts

```diff
@@ -149,6 +151,24 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 		return content;
 	}
 
+	// Only replace vscode-resource with file when in the same (or a sub) directory
+	// This matches Jupyter Notebook viewer behavior
+	private convertVsCodeResourceToFileInSubDirectories(htmlContent: string): string {
+		let htmlContentCopy = htmlContent;
+		while (htmlContentCopy.search('(?<=img src=\"vscode-resource:)') > 0) {
+			let pathStartIndex = htmlContentCopy.search('(?<=img src=\"vscode-resource:)');
+			let pathEndIndex = htmlContentCopy.slice(pathStartIndex).indexOf('\" ') + pathStartIndex;
```

Comment:
Questions:
- Could there be a case where someone escapes this as part of the path, e.g. `\\" ` and this matches incorrect? Am I just being paranoid?
- Slicing large text could be super inefficient? Looking at [w3 docs](https://www.w3schools.com/jsref/jsref_indexof.asp), I think you can pass in the start position to search from and avoid this issue. Can you do that instead? It should be a lot more efficient.
```
			let pathEndIndex = htmlContentCopy.indexOf('\" ', pathStartIndex);
```

Reply:
Addressed the second bullet. For the first one, someone would have to have a folder/file name that is literally called folderAbove\" blah"

i.e. the folder would have to start with a quote, followed by a space.

I'm not convinced this is a mainline scenario, even if it has a tiny chance at being a bug.

Reply:
Sounds good, thanks